.github/actions/setup-env/action.yml

```diff
@@ -41,11 +41,11 @@ runs:
         pip install -U pip
         pip install pipenv pdm
 
-    # Node.js
     - name: Setup pnpm
       if: inputs.setup_nodejs == 'true'
-      uses: pnpm/action-setup@v3
-      # Automatically reads `packageManager` field from root `package.json`.
+      shell: bash
+      # Corepack automatically reads `packageManager` field from root `package.json`.
+      run: corepack enable pnpm
```

Comment:
👍 TIL that `corepack enable` takes the name of the package manager as an argument!

Reply:
It doesn't need to have the package name, but there were warnings in some places without it.